sdk/src/client/api/types.rs

```diff
@@ -169,7 +170,7 @@ pub struct RemainderData {
     /// The remainder output
     pub output: Output,
     /// The chain derived from seed, for the remainder addresses
-    pub chain: Option<Chain>,
+    pub chain: Option<Bip44>,
```

Comment:
I wonder if we should put serde_with everywhere? In case there are places we messed up and stored a real type instead of a DTO

Reply:
Well we would need to be sure to migrate it if that's the case